src/verifiable/verifiable-payid.ts

```diff
@@ -10,6 +10,9 @@ import OKPKey = JWK.OKPKey
 
 export enum AddressDetailsType {
```

Comment:
Might be good to leave a comment to swap this out for a types library once we have it.